packages/mongo-nav/src/org-nav/OrgNav.spec.tsx

```diff
@@ -146,12 +165,30 @@ describe('packages/mongo-nav/src/org-nav', () => {
 
   describe('when rendered as an admin with an active preferences nav', () => {
     beforeEach(() =>
-      renderComponent({ activeNav: 'userSettings', admin: true }),
+      renderComponent({
+        activeNav: 'userSettings',
+        admin: true,
+      }),
     );
+
     testForPaymentStatus(false);
+    testForVersion(false);
 
     Object.keys(linkNamesToUrls).forEach(linkName =>
       testForNavLink(linkName, ['allClusters', 'admin'].includes(linkName)),
     );
   });
+
+  describe('when rendered onPrem', () => {
```

Comment:
We now additionally need to test:
- the expected user menu is rendered by default
- the on prem user menu is rendered when on prem is true

And I think that means adding testing for the contents of the on prem menu if there are any conditional things rendered / if urls are overwritten, etc.

Reply:
👍